tests/marqo_test.py

```diff
@@ -112,6 +120,7 @@ def clear_indexes(self, indexes: List[MarqoIndex]):
             self.clear_index_by_name(index.schema_name)
 
     def clear_index_by_name(self, index_name: str):
+        """Not that the index_name here is the schema_name, not the index name"""
```

Comment:
nit. Typo Not->Note. But should we just rename the parameter to schema_name to be clear?